examples/platform/efr32/init_efrPlatform.cpp

```diff
@@ -57,6 +57,7 @@ extern "C" {
 #include "sl_system_init.h"
 
 void initAntenna(void);
+void wifi_board_init(void);
```

Comment:
why is the prototype here? Shouldn't it be in a header file where your wifi source code is